lib/role_model/class_methods.rb

```diff
@@ -41,21 +41,20 @@ def roles(*roles)
       opts = roles.last.is_a?(Hash) ? roles.pop : {}
       self.valid_roles = roles.flatten.map(&:to_sym)
       unless (opts[:dynamic] == false)
-        self.define_dynamic_queries(self.valid_roles)
+        self.define_dynamic_queries(self.valid_roles, opts[:prefix])
       end
     end
-    
-    # Defines dynamic queries for :role
-    #   #is_<:role>?
+
+    # Defines dynamic queries for :role. Providing a prefix is optional.
     #   #<:role>?
+    #   #<:role>=(true|false)
     #
-    # Defines new methods which call #is?(:role)
-    def define_dynamic_queries(roles)
+    # Defines new methods which call #is?(:role) and roles.(add|remove)
+    def define_dynamic_queries(roles, prefix=nil)
```

Comment:
I suggest defaulting `prefix=''` to make clearer that it is a String, even though `nil.to_s == ''`.